### PR TITLE
[+] Explicit Path ID: Clarify use of MAX_PATHS

### PR DESCRIPTION
**Pull request by Yanmei-Liu**
Solving issue #298.

### CODE REVIEW

**Review by marten-seemann — COMMENTED** [review]
I'm not a fan of _negotiating_ the value. I'd much rather have peers _declare_ the maximum path ID they're willing to accept. This requires path IDs to be unidirectional.

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -324,8 +324,21 @@ Use of the "preferred address" is considered as a migration event
...
 Endpoints use PATH_ABANDON frame to inform the peer of the retirement of associated 
-Path Identifier. When there is not enough unused Path Identifiers, endpoints SHOULD
+Path Identifier. 
+
+When there is not enough unused Path Identifiers, endpoints SHOULD
 send MAX_PATHS frame to inform the peer that new Path Identifiers are available.
+For example, the server send MAX_PATHS to raise the maximum path number that it is willing to accept,
+the client sends MAX_PATHS to raise the maximum path number that it is willing to create.
+
+Endpoints MUST NOT issue new Connection ID with Path ID larger than the mininum of MAX_PATHS
```
Comment (qdeconinck) [inline draft-ietf-quic-multipath.md:334]:
```suggestion
Endpoints MUST NOT issue new Connection ID with Path ID larger than the minimum of MAX_PATHS
```

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -324,8 +324,21 @@ Use of the "preferred address" is considered as a migration event
...
+announced by both endpoints. For example, the server sends MP_NEW_CONNECTION_ID frame containing 
+Path ID up to the mininum of MAX_PATHS issued by client and server. The client sends 
+MP_NEW_CONNECTION_ID frame containing Path ID up to the mininum of MAX_PATHS issued by each sides.
```
Comment (qdeconinck) [inline draft-ietf-quic-multipath.md:337]:
```suggestion
MP_NEW_CONNECTION_ID frame containing Path ID up to the minimum of MAX_PATHS issued by each sides.
```

**Comment by mirjak** [comment]
Closing this as obsoleted by #331

**Pull request closed without merging**